wikidataQuery.py

```diff
@@ -10,26 +10,25 @@
                   'play': 'instrument'}
 
 # List of w-words, feel free to add any words I forgot
-w_words_list = ['What', 'Who', 'When', 'Where', 'Why', 'How', 'Which']
```

Comment:
I'd like to keep this one if possible. Maybe even move it to either `syntacticAnalysis.py` or `settings.py`

Reply:
I'm not sure where it would be useful though. If you'll need it though, you can always modify the dict a bit & then move it.

Reply:
Sure, that's fine as well. Somewhere in the `syntacticAnalysis.py` file I have a statement that checks for the 'which' word right now because that was what the example question had, but I think it can be generalised to all these words. So that's what I needed it for.